### PR TITLE
[FLINK-20310][doc] Add documentation for Debezium, Canal, Raw support for Filesystem connector

### PR DESCRIPTION
**Pull request by JingsongLi**
Update format index and filesystem.
image
image

### CODE REVIEW

File: docs/dev/table/connectors/formats/index.zh.md [hunk 1]
```diff
@@ -58,10 +58,12 @@ Flink 支持以下格式：
         <tr>
          <td><a href="{% link dev/table/connectors/formats/debezium.zh.md %}">Debezium CDC</a></td>
          <td><a href="{% link dev/table/connectors/kafka.zh.md %}">Apache Kafka</a></td>
+         <td><a href="{% link dev/table/connectors/filesystem.zh.md %}">Filesystem</a></td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/index.zh.md:61]: It's not a new `<td>` column, but just add Filesystem link in the column.

Comment (JingsongLi) [reply docs/dev/table/connectors/formats/index.zh.md:61]: Oh, sorry, get it.

**Comment by JingsongLi** [comment]
Updated picture

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**